src/System.Data.SqlClient/src/System/Data/SqlClient/TdsParserStateObject.cs

```diff
@@ -63,8 +63,8 @@ internal abstract class TdsParserStateObject
         // Packet state variables
         internal byte _outputMessageType = 0;                   // tds header type
         internal byte _messageStatus;                               // tds header status
-        internal byte _outputPacketNumber = 1;                   // number of packets sent to server
-                                                                 // in message - start at 1 per ramas
+        internal byte _outputPacketNumber = 1;                   // number of packets sent to server in message - start at 1 per ramas
```

Comment:
What does ramas stand for/means?

Reply:
And I just saw this already existed in the code.

Reply:
I assumed it was the name of the guy on the sql server team that the author asked for help 😁